deploy/004-omnichain-remote.ts

```diff
@@ -90,42 +93,18 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     autoMine: true,
   });
 
-  const bridge = (await ethers.getContractAt(
+  const omnichainGovernanceExecutor = (await ethers.getContractAt(
     "OmnichainGovernanceExecutor",
     OmnichainGovernanceExecutor.address,
     ethers.provider.getSigner(deployer),
   )) as OmnichainGovernanceExecutor;
 
-  const normalTimelock = await deploy("Timelock_Normal", {
-    contract: "Timelock",
-    from: deployer,
-    args: [bridge.address, timelockDelays[hre.network.name].NORMAL],
-    log: true,
-    autoMine: true,
-  });
-
-  const fasttrackTimelock = await deploy("Timelock_FastTrack", {
-    contract: "Timelock",
-    from: deployer,
-    args: [bridge.address, timelockDelays[hre.network.name].FAST_TRACK],
-    log: true,
-    autoMine: true,
-  });
-
-  const criticalTimelock = await deploy("Timelock_Critical", {
-    contract: "Timelock",
-    from: deployer,
-    args: [bridge.address, timelockDelays[hre.network.name].CRITICAL],
-    log: true,
-    autoMine: true,
-  });
-
   const OmnichainExecutorOwner = await deploy("OmnichainExecutorOwner", {
     from: deployer,
     args: [OmnichainGovernanceExecutor.address],
     contract: "OmnichainExecutorOwner",
     proxy: {
-      owner: normalTimelock.address,
+      owner: deployer,
```

Comment:
Setting the owner as the timelock won't work because in the initialization the Ownable init function will fail. Instead we are transferring ownership to the timelock after running the required commands on line 144 [here](https://github.com/VenusProtocol/governance-contracts/pull/41/files#diff-c2651d8481480f71690b29a190d4f5b97cbe9be91c20a8f3577491461d9c2a2bR144)

Reply:
Isn't it is a way for configure owner of `DefaultProxyAdmin` [here](https://github.com/wighawag/hardhat-deploy#:~:text=empty%20ETH%20payment.-,OpenZeppelinTransparentProxy,-%3A%20Use%20Openzeppelin%20Transparent)